src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1177,6 +1177,10 @@
   <Node Name="BoundDagPropertyEvaluation" Base="BoundDagEvaluation">
     <Field Name="Property" Type="PropertySymbol" Null="disallow"/>
   </Node>
+  <Node Name="BoundDagIndexEvaluation" Base="BoundDagEvaluation">
```

Comment:
>BoundDagIndexEvaluation [](start = 14, length = 23)

Do we need to customize the `Display` property for the new bound nodes (in `Formatting.cs`)?

Reply:
No, that is for nodes that extend `BoundExpression`.